var/spack/repos/builtin/packages/py-stashcp/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyStashcp(PythonPackage):
+    """Stashcp uses geo located nearby caches in order to copy from the OSG
+    Connect's stash storage service to a job's workspace on a cluster."""
+
+    homepage = "https://github.com/opensciencegrid/StashCache"
+    pypi = "stashcp/stashcp-6.1.0.tar.gz"
+
+    maintainers("wdconinc")
+
+    version("6.1.0", sha256="40484b40aeb853eb6a5f5472daf533a176d61fa6ab839cd265ea0baa3fe63068")
+
+    depends_on("py-setuptools", type="build")
+
+    depends_on("py-urllib3", type=("build", "run"))
```

Comment:
Can you add a comment where this is coming from because this dependency is not specified in `setup.py`. E.g that it is imported in `stashcp.__init__.py`